led_raspberrypi_demo/README.md

```diff
@@ -0,0 +1,54 @@
+# Light Mapper
+
+Light Mapper contains code to control an LED light connected to a raspberry Pi through gpio.
+
```

Comment:
Can we have following structure in the README of each demo ?
Demo
         | Description (detailed description of the demo)
         | Prerequisites/SW & HW used (should specify kubeedge version used, RPi model, go version, etc)
         | Steps to reproduce
         | Additional Usecases (similar usecases if any)